### ISSUE TITLE
NTR: Lung imaging: pulmonary perfusion defects (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8919]
New term request Lung imaging: pulmonary perfusion defects (elthat_100722020936): Arteriography, CT angiography, or ventilation perfusion scan; hypoperfusion  UMLS: 4028621

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000412
- label: Pulmonary perfusion defect
- parents: [Abnormality of pulmonary circulation (HP:0030875)](https://hpo.jax.org/app/browse/term/HP:0030875)
- pmid: PMID:33232086
- def: Lack of blood perfusion in a specific area of the lung. Pulmonary ventilation (V) and Perfusion (Q) scan, also known as lung V/Q scan, is a nuclear test that uses the perfusion scan to delineate the blood flow distribution and the ventilation scan to measure airflow distribution in the lungs.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8919
- synonyms:
